apps/src/templates/VersionHistory.jsx

```diff
@@ -183,6 +185,12 @@ export default class VersionHistory extends React.Component {
               versionId={version.versionId}
               lastModified={new Date(version.lastModified)}
               isLatest={version.isLatest}
+              isViewingVersion={
```

Comment:
(minor) I find the name `isViewingVersion` a little confusing, maybe `isCurrentVersion` or `isSelectedVersion`?

Reply:
Done!